app/frontend/translations.ts

```diff
@@ -6,6 +6,9 @@ import {Lovelace, CryptoProviderFeature} from './types'
 
 const {ADALITE_MIN_DONATION_VALUE} = ADALITE_CONFIG
 
+const ledgerConnectError =
```

Comment:
ledgerTroubleshootingSuggestion is imho a more accurate name as it's not an error per se